maker/src/main.rs

```diff
@@ -123,6 +127,20 @@ async fn main() -> Result<()> {
     let db =
         sqlite_db::connect(data_dir.join("maker.sqlite"), opts.ignore_migration_errors).await?;
 
+    let blocked_peers_path = data_dir.join("blocked_peers");
+    let blocked_peers = match load_blocked_peers(&blocked_peers_path).await {
+        Ok(loaded) => loaded,
+        Err(error) => {
+            tracing::error!(
+                ?blocked_peers_path,
+                %error,
+                "Error loading blocked peers list; ignoring and allowing all connections",
+            );
+
+            HashSet::default()
+        }
+    };
```

Comment:
While it is technically rolling out own format, it's just lines of peer ids, so this doesn't add any maintenance burden and isn't confusing imo. It might be beneficial if we add e.g comments later.

Reply:
Done now :)